app/javascript/groceries/components/check_in_items_list.vue

```diff
@@ -4,7 +4,7 @@ section(v-if="items.length > 0")
 
   ul.check-in-items-list.mb-2
     li.flex.items-center.mb-2(
-      v-for='(item, index) in items'
+      v-for='item in items'
```

Comment:
TypeScript [caught](https://github.com/davidrunger/david_runger/actions/runs/9865525548/job/27242625078?pr=4577#step:10:239) some unused variables (even in templates; in fact, exclusively in templates, I guess maybe because ESLint wasn't going there before), presumably because we enabled the `noUnusedLocals` TypeScript compiler option.